src/app/join-us/page.tsx

```diff
@@ -1,32 +1,166 @@
+        <Box sx={{ width: '50%' }}>
+          <Image
+            src="https://source.unsplash.com/random?wallpapers"
+            alt="Description"
+            width={500}
+            height={500}
+            style={{ width: '100%' }}
+          />
+        </Box>
+        <Box sx={{ width: '50%', marginX: '2em' }}>
+          <Typography className="sub-header" variant="h5">
+            Our Mission
+          </Typography>
+          <Typography variant="body2" paragraph>
+            At OpenBC, we&apos;re passionate about leveraging your unique
+            abilities for the advancement of our province.
+            <strong>
+              Contribute to Open Source Projects for BC, from Anywhere, with
+              OpenBC.
+            </strong>
+          </Typography>
+        </Box>
+      </Box>
+      <Box sx={{ marginY: '4em' }}>
+        <Typography className="sub-header" variant="h5">
+          Visions
+        </Typography>
+        <Typography variant="body2" paragraph>
+          Quis neque, eu et ipsum amet, vel et. Varius integer enim pellentesque
+          ornare pharetra faucibs arcu. Mauris blandit egestas nibh. eu et ipsum
+          amet, vel et. Varius integer enim pellentesque ornare pharetra faucibs
+          arcu. Mauris blandit egestas nibh.
+        </Typography>
+      </Box>
+      <Box sx={{ width: '40%', alignSelf: 'center' }}>
+        <Box sx={{ display: 'flex', justifyContent: 'space-between' }}>
+          <Typography variant="h5">Volunteers</Typography>
+          <Typography variant="h5">Be A Partner</Typography>
+        </Box>
+        <Divider sx={{ borderRadius: 1 }} />
+      </Box>
+      <Box className="sub-container-2-col">
+        <Box sx={{ width: '50%' }}>
+          <Typography className="sub-header" variant="h5">
+            <strong>
+              Ready to contribute to British Columbia through Open source
+              projects?
+            </strong>
+          </Typography>
+          <Typography variant="body2" paragraph>
+            Join us at OpenBC and become a part of our active and welcoming
+            community.
+          </Typography>
+        </Box>
+        <Box sx={{ width: '50%' }}>
+          <Box sx={{ display: 'flex' }}>
+            <Paper className="square">
+              <Box
+                sx={{
+                  display: 'flex',
+                  alignItems: 'center',
+                  justifyContent: 'center',
+                  height: '100%',
+                }}
+              >
+                <Typography>1</Typography>
+              </Box>
+            </Paper>
+            <Typography sx={{ marginLeft: '1em' }} variant="h5">
+              Tell Us About Yourself
+            </Typography>
+          </Box>
+          <Typography sx={{ marginLeft: '4em' }} variant="body2" paragraph>
+            Fill out our <u>2 minute form</u> so we can help match you with
+            opportunities
+          </Typography>
```

Comment:
Admittedly this is possibly a nitpick, but I wonder for maintainability (i.e. if we ever want this to be mobile friendly), would it be possible to move some of the styling props here to the `.scss` file?

So for example:

```
             <Box
                sx={{
                  display: 'flex',
                  alignItems: 'center',
                  justifyContent: 'center',
                  height: '100%',
                }}
              >
```

```
<Typography sx={{ marginLeft: '1em' }} variant="h5">
```

```
<Typography sx={{ marginLeft: '4em' }} variant="body2" paragraph>
```

Could these CSS styles be moved to the `.scss` file, and then the appropriate classes called in the subsequent similar `Box` and `Typography` components below?